### DISCUSSION TITLE
Install on linux distros

### DISCUSSION THREAD

**Discussion by sebastian-blandon** [posit-dev/positron#3675] [Q&A] [answered]
Hello there.

Are there any instructions for installing Positron on other linux distros, please? I didn't find any DPKGBUILD file, my OS is Manjaro Linux. Thank you

**Comment by jmcphers** [comment]
Sorry, we only have builds for Debian-based Linux distributions today. We'll be adding support for other popular Linux distributions before we release.

**Reply by aalexandersson** [reply]
Please add support for [Flatpak](https://flatpak.org/). Then it does not matter if you use Debian or Manjaro or whatever other Linux OS. Not having Flatpak is perhaps the main weaknesses in `RStudio` compared with VS Code, as discussed in `RStudio` [issue 3079](https://github.com/rstudio/rstudio/issues/3079). 

Update: Though Flatpak installation would be nice to have, I do not think it is not critical. There is no official Flatpak build for Visual Studio Code. For the official builds, see [https://code.visualstudio.com/download](https://code.visualstudio.com/download). 

Instead, what I think is critical is Positron support for dev containers, preferably enabled by default and with Docker at as the default backend. But that is a separate issue than discussed here.

**Comment by chuxinyuan** [comment, marked as answer]
<https://aur.archlinux.org/packages/positron-ide-devel-bin>

**Reply by sebastian-blandon** [reply]
Thank you so much!

**Comment by m-fariz-a** [comment]
In general, arguably most top editor and IDE for Linux are also packaged in Snaps than Flatpak e.g Visual Studio Code or other JetBrains Products. I do not know the exact reason why, but I do not mind either it is in Snaps or Flatpak. Each packaging format has its own pros and cons.

I hope `Positron` will be getting there too. :)

**Reply by petetronic** [reply]
Thanks @m-fariz-a, we have had some initial discussions on this, but I don't see an actual issue logged. I'll log one now to capture this request.

**Reply by petetronic** [reply]
Started https://github.com/posit-dev/positron/issues/4721 to track this topic.
